collections/guides/use/run-your-own-code-in-a-reactor.md

```diff
@@ -0,0 +1,345 @@
+- `tokenize`: sensitive data that will be tokenized before returning it in the response
+
+Exercise caution when returning plaintext data within the `raw` portion of the response - the systems that invoke your 
+Reactor will receive this raw data in the HTTP response and returning sensitive data in plaintext could result in 
+these systems being pulled into compliance scope.
+
+While you have the freedom to author your Reactor Formula in any way you wish, we recommend that you strive to keep 
+all sensitive data isolated within Basis Theory's secure and compliant infrastructure.
+We generally recommend that any potentially sensitive output be tokenized in order to shield your systems.
+
+### Importing Dependencies
+
+You are able to import dependencies within Reactor Formulas. 
+For example, `node-fetch` can be imported into your code and used to make an HTTP request:
+
+```js
+module.exports = async function (req) {
+  const { card_number } = req.args;
+  const { API_KEY } = req.configuration;
+  const fetch = require('node-fetch');
+
+  const res = await fetch(`https://api.company.com/`, {
+    headers: { 
+      'X-API-KEY': API_KEY,
+      'Content-Type': 'application/json'
+    },
+    method: 'POST',
+    body: JSON.stringify({ cardNumber: card_number })
+  });
+    
+  return { ... };
+};
+```
+
+In order to ensure the security and integrity of the Reactor execution environment, only approved NPM packages can be used within Reactor Formulas.
+If you are interested in using a specific NPM package within your custom code, please 
+[submit a request](https://support.basistheory.com/hc/requests/new?tf_subject=Request%20NPM%20Package%20for%20Private%20Reactor%20Formulas&amp;tf_description=Let%20us%20know%20what%20package%20you'd%20like%20to%20use&amp;tf_priority=normal) 
+to have this package reviewed and whitelisted. 
+
+### Error Handling
+
+Many third party APIs or SDKs return errors in unique ways, so Reactor Formulas are expected to catch and handle any errors that occur during execution. 
+Any errors that occur during the execution of your Reactor Formula should be caught, translated into a standard Basis Theory [Reactor error](https://github.com/Basis-Theory/basistheory-reactor-formulas-sdk-js/tree/master/src/errors),
+and this standardized error rethrown from the formula.
+
+The standard Basis Theory [Reactor errors](https://github.com/Basis-Theory/basistheory-reactor-formulas-sdk-js/tree/master/src/errors) 
+are defined within the NPM package [@basis-theory/basistheory-reactor-formulas-sdk-js](https://www.npmjs.com/package/@basis-theory/basis-theory-reactor-formulas-sdk-js),
+which is available to import from within your Reactor Formulas.
+These standard error types will be translated by the Reactor execution environment into [HTTP errors](https://docs.basistheory.com/#errors-reactor-errors)
+returned from the Basis Theory API. Any other uncaught exceptions thrown from a Reactor Formula will be returned as an HTTP `500` error.
```

Comment:
Do we want to call out that the original error message will be lost and not returned in the generic 500 response in this scenario?